libs/frontend/feature-render-builder/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -203,24 +203,6 @@ export const BuilderDndComponent = defineComponent({
         }
       }
 
-      if (
```

Comment:
@abemscac I moved this to `use-drag-drop` because it seemed more efficient, and easier to follow what's going on with the code. I think there may be a situation where it causes the DND overlay to remain on screen when it shouldn't, but I can't reproduce it. That problem may be solved by simplifying further and moving the code that clears `runtimeContext.buildDndState` back to `use-drag-drop`.

Reply:
Got it, thanks for letting me know!